clang/test/CodeGenCUDA/cuda-builtin-vars.cu

```diff
@@ -6,21 +6,21 @@
 __attribute__((global))
 void kernel(int *out) {
   int i = 0;
-  out[i++] = threadIdx.x; // CHECK: call noundef i32 @llvm.nvvm.read.ptx.sreg.tid.x()
-  out[i++] = threadIdx.y; // CHECK: call noundef i32 @llvm.nvvm.read.ptx.sreg.tid.y()
-  out[i++] = threadIdx.z; // CHECK: call noundef i32 @llvm.nvvm.read.ptx.sreg.tid.z()
+  out[i++] = threadIdx.x; // CHECK: call noundef {{.*}} i32 @llvm.nvvm.read.ptx.sreg.tid.x()
```

Comment:
This change appears to be causing the test to fail on some bots where there are no extra arguments in this location, causing a single space ` ` to not match the expected double space `  `. You can probably fix this by just removing either the leading or trailing space that you added for each of these.

Bot failure:
https://lab.llvm.org/buildbot/#/builders/139/builds/67000
```

 out[i++] = threadIdx.x; // CHECK: call noundef {{.*}} i32 @llvm.nvvm.read.ptx.sreg.tid.x()
                                   ^
<stdin>:17:53: note: scanning from here
define dso_local void @_Z6kernelPi(ptr noundef %out) #0 {
                                                    ^
<stdin>:23:7: note: possible intended match here
 %0 = call noundef i32 @llvm.nvvm.read.ptx.sreg.tid.x()
      ^
```

Reply:
@AlexMaclean I also see this problem on some internal test machines. It seems suspicious - is there some nondeterminism? Or is there a good reason why some machines would not add the range metadata here???

Reply:
I see now that it fails (deterministically) if the NVPTX target is not being built.

Reply:
@AlexMaclean can you either fix the test or revert the change so that we can get the failing bots green again please?

Reply:
https://github.com/llvm/llvm-project/pull/94639